### ISSUE TITLE
Poor results

### ISSUE THREAD

**Issue by adarc8** [wilmsm/lightweightbraindiff#1]
Hey guys, great work!
I managed to run training, but the results are low quality.. Here is a random 2d slice from the 3d generated scan:

![Image](https://github.com/user-attachments/assets/b9cf0eec-b0d2-48c6-840c-0896268a08e4)

Did you use this code for the paper? maybe any additional code blocks?
its pretty different base code then [wdm-3d repo](https://github.com/pfriedri/wdm-3d)

**Comment by wilmsm** [comment]
Thanks for your feedback! Yes, this is the code used for training our model and to generate the images in the paper. Depending on the size of your training data set, you may have to change the training hyper parameters by for example increasing the number of epochs used etc..

Matthias

**Comment by adarc8** [comment]
Thanks for your answer :)
Do u by any chance have a loss function graph so i can compare if mine make sense?

Here are my results after 30 Hour training (50 epochs of 30k 3d scans)
L2 loss reached 3e-4 and 4.5e-4 (train val)
![Image](https://github.com/user-attachments/assets/7277a2d3-bfe6-491e-a855-b1eee5c33a87)

![Image](https://github.com/user-attachments/assets/edd84e42-7ea6-47a9-bca1-d728f3e487bf)

My preprocess is slightly different but I resize to same dimensions as you guys used
(30k data is UkBioBank + others)

**Comment by wilmsm** [comment]
Unfortunately, I don't have a plot to share right now. Hard to tell what's going wrong in your setup. Seems that the loss is still improving, though. Also, your pre-processing appears to be totally different from ours as your last image appears to include the skull/non-brain tissue.

Matthias

**Comment by adarc8** [comment]
OK cool thanks!
what other preprocessing you guys did? did u made any registration?

**Comment by wilmsm** [comment]
Yes, you can find more details in the paper.
